src/libs/Navigation/AppNavigator/Navigators/FullScreenNavigator.tsx

```diff
@@ -20,17 +39,19 @@ function FullScreenNavigator() {
 
     return (
         <View style={styles.rootNavigatorContainerStyles(isSmallScreenWidth)}>
-            <RootStack.Navigator>
+            <RootStack.Navigator screenOptions={screenOptions.fullScreenInner}>
                 <RootStack.Screen
                     name={SCREENS.WORKSPACE.INITIAL}
                     options={screenOptions.homeScreen}
                     getComponent={loadWorkspaceInitialPage}
                 />
-                <RootStack.Screen
-                    name={SCREENS.WORKSPACES_CENTRAL_PANE}
```

Comment:
I think we also need to remove `WORKSPACES_CENTRAL_PANE` from `SCREENS.ts` as there is no usage for it now